integrations/huggingface-transformers/tutorials/question-answering/question_answering-cli.md

```diff
@@ -0,0 +1,372 @@
+# Question Answering: Sparse Transfer Learning with the CLI
+
+In this example, you will sparse transfer learn a 90% pruned BERT model onto some extraction question answering datasets using SparseML's CLI.
+
+### **Sparse Transfer Learning Overview**
+
+Sparse Transfer Learning is very similiar to the typical transfer learing process used to train NLP models, where we fine-tune a pretrained checkpoint onto a smaller downstream dataset. With Sparse Transfer Learning, we simply start the fine-tuning process from a pre-sparsified checkpoint and maintain sparsity while the training process occurs.
```

Comment:
With Sparse Transfer Learning, we start the fine-tuning process from a pre-sparsified checkpoint and maintain sparsity during the training process.